test/functional/feature_bip68_sequence.py

```diff
@@ -92,16 +91,12 @@ def run_test(self):
     # the first sequence bit is set.
     def test_disable_flag(self):
         # Create some unconfirmed inputs
-        new_addr = self.nodes[0].getnewaddress()
-        self.nodes[0].sendtoaddress(new_addr, 2) # send 2 BTC
-
-        utxos = self.nodes[0].listunspent(0, 0)
-        assert len(utxos) > 0
+        self.wallet.send_self_transfer(from_node=self.nodes[0])
 
-        utxo = utxos[0]
+        utxo = self.wallet.get_utxo()
```

Comment:
This may not be unconfirmed. If you need the unconfirmed one from above you can fetch is with `["new_utxo"]`.

Unrelated, I wonder why this needs to be unconfirmed

Reply:
Should I change the code to match the origin behavior, which is unconfirmed UTXO?

```diff
diff --git a/test/functional/feature_bip68_sequence.py b/test/functional/feature_bip68_sequence.py
index e53e177ce..e14867dc3 100755
--- a/test/functional/feature_bip68_sequence.py
+++ b/test/functional/feature_bip68_sequence.py
@@ -91,9 +91,7 @@ class BIP68Test(BitcoinTestFramework):
     # the first sequence bit is set.
     def test_disable_flag(self):
         # Create some unconfirmed inputs
-        self.wallet.send_self_transfer(from_node=self.nodes[0])
-
-        utxo = self.wallet.get_utxo()
+        utxo = self.wallet.send_self_transfer(from_node=self.nodes[0])["new_utxo"]
 
         tx1 = CTransaction()
         value = int((utxo["value"] - self.relayfee) * COIN)
```